home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1791,6 +1916,66 @@ components:
           maxLength: 13
           type: string
           example: "+380966617305"
+    CooperationDataScope:
+      type: string
+      enum:
+        - ALL
+        - ADRESS
+        - APARTMETS
+        - OWNERS
+    TaskStatus:
+      type: string
+      enum:
+        - PENDING
+        - ERROR
+        - PROCESSING
+        - ACCEPTED
+        - DECLINED
+    TaskType:
+      type: string
+      enum:
+        - IMPORT
+        - EXPORT
+    ReadTask:
+      type: object
+      properties:
+        id:
+          type: integer
+          format: int64
+          example: 1
+        type:
+          $ref: '#/components/schemas/TaskType'
+        status:
+          $ref: '#/components/schemas/TaskStatus'
+    ReadReport:
+      type: object
+      properties:
+        houses_amount:
+          type: integer
+          format: int64
+          example: 1
+        apartments:
+          type: integer
+          format: int64
+          example: 1
+        invitations:
+          type: integer
+          format: int64
+          example: 1
+    ReadXlsxFile:
```

Comment:
This schema is used both as a payload for importing a file and as response content when exporting a file. I think that it would be more correct to rename it, for example, to "XlsxFilePayload".

Reply:
What about :
- ReadTaskSpreadsheet
- UploadTaskSpreadsheet
- ReadTeamplateTaskSpreadsheet - BTW, am I missed the template resource endpoint?